server/routes/index.js

```diff
@@ -14,6 +15,9 @@ module.exports = (app) => {
   // drills
   app.post('/api/sessions/:session_id/drills', drillsController.create);
 
+  app.get('/api/actions', actionsController.GetAll);
+  app.post('/api/new/actions', actionsController.NewAction);
```

Comment:
You don't need to specify the route /new/. You just use the different method on the same endpoint. IE just having the post